src/api/forms/service.js

```diff
@@ -189,6 +197,55 @@ export async function updateDraftFormDefinition(formId, definition, author) {
   }
 }
 
+/**
+ * @param {string} formId - ID of the form
+ * @param {Partial<FormMetadataInput>} formUpdate - full JSON form definition
+ * @returns {Promise<string>}
+ */
+export async function updateFormMetadata(formId, formUpdate) {
+  logger.info(`Updating form metadata for form ID ${formId}`)
+
+  try {
+    // Get the form metadata from the db
+    const form = await getForm(formId)
+
+    if (form.live && 'title' in formUpdate) {
+      throw Boom.badRequest(
+        `Form with ID '${formId}' is live so 'title' cannot be updated`
+      )
+    }
+
+    let updatedSlug
+
+    const updatedForm = {
+      slug: form.slug,
+      ...formUpdate
+    }
+
+    if (formUpdate.title) {
+      updatedSlug = slugify(formUpdate.title)
+    } else {
+      updatedSlug = form.slug
+    }
+
+    const session = client.startSession()
+
+    try {
+      // Update the form metadata
+      await formMetadata.update(formId, { $set: updatedForm }, session)
+      logger.info(`Updated form metadata for form ID ${formId}`)
+
+      return updatedSlug
+    } finally {
+      await session.endSession()
+    }
```

Comment:
since we're not using the session, try dropping this try/catch and drop the `session` parameter